deploy/deployment/openshift/objects/devworkspaceroutings.controller.devfile.io.CustomResourceDefinition.yaml

```diff
@@ -19,7 +19,20 @@ spec:
     singular: devworkspacerouting
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
```

Comment:
Seems our [gitattributes](https://github.com/devfile/devworkspace-operator/blob/main/.gitattributes) is not up-to-date

Reply:
:thinking:

Reply:
Figured it out https://github.com/devfile/devworkspace-operator/pull/389